paasta_tools/automatic_rollbacks/slack.py

```diff
@@ -59,29 +59,37 @@ def get_slack_blocks_for_deployment(
     if slo_watchers is not None and len(slo_watchers) > 0:
         num_failing = len([w for w in slo_watchers if w.failing])
         if num_failing > 0:
-            blocks.append({
-                "type": "section",
-                "text": {
-                    "type": "mrkdwn",
-                    "text": f":alert: {num_failing} of {len(slo_watchers)} SLOs are failing",
-                },
-            })
+            slo_text = f":alert: {num_failing} of {len(slo_watchers)} SLOs are failing"
         else:
-            blocks.append({
-                "type": "section",
-                "text": {
-                    "type": "mrkdwn",
-                    "text": f":ok_hand: All {len(slo_watchers)} SLOs are currently passing.",
-                },
-            })
+
+            num_unknown = len([w for w in slo_watchers if w.bad_before_mark is None or w.bad_after_mark is None])
```

Comment:
This block of code is complex enough to justify its own function.
It would be a little easier to test I think if that was the case